bitnami/jenkins/README.md

```diff
@@ -196,6 +196,7 @@ s
 | `kubeVersion`            | Override Kubernetes version                                                             | `""`            |
 | `nameOverride`           | String to partially override common.names.fullname                                      | `""`            |
 | `fullnameOverride`       | String to fully override common.names.fullname                                          | `""`            |
+| `replicaCount`           | Number of containers                                                                    | `1`             |
```

Comment:
Do not manually change the table.
Generate it like described: https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#documentation-requirements